modules/drivers/mcp230/mcp230.js

```diff
@@ -32,138 +32,104 @@ const PULLUPS_8  = 0b00000000;
       }
     }
 
     // If the value of pullups does not match the default,
     // then set the user defined pullups configuration
-    if ((pullups !== (offset ? PULLUPS_16 : PULLUPS_8))) {
-      this.writeByte(GPPU, pullups);
+    if ((pullups !== (this.offset ? PULLUPS_16 : PULLUPS_8))) {
+      this.writeByte(this.GPPU, pullups);
 
-      if (offset) {
-        this.writeByte(GPPU + offset, pullups >> 8);
+      if (this.offset) {
+        this.writeByte(this.GPPU + this.offset, pullups >> 8);
       }
     }
 
-    for (let pin = 0; pin < pins; pin++) {
+    for (let pin = 0; pin < this.length; pin++) {
       this[pin] = new Pin({ pin, expander: this });
     }
 
-    this.length = pins;
-    this.offset = offset;
-    this.reg = Object.freeze(reg);
-
     Object.freeze(this);
   }
 
-  * [Symbol.iterator]() {
-    let i = 0;
-    while (i < this.length) {
-      yield this[i];
-      i++;
-    }
-  }
-
   bankWrite(state) {
-    const { IODIR, GPIO } = this.reg;
-
     if (this.offset) {
       // Read IODIR state
-      let iodir = this.readWord(IODIR);
+      let iodir = this.readWord(this.IODIR);
 
       // Set IODIR state to OUTPUT
-      this.writeWord(IODIR, 0x0000);
+      this.writeWord(this.IODIR, 0x0000);
 
       // Write GPIO
-      this.writeWord(GPIO, state);
+      this.writeWord(this.GPIO, state);
 
       // Restore previous IODIR state
-      this.writeWord(IODIR, iodir);
+      this.writeWord(this.IODIR, iodir);
     } else {
       // Read IODIR state
-      let iodir = this.readByte(IODIR);
+      let iodir = this.readByte(this.IODIR);
 
       // Set IODIR state to OUTPUT
-      this.writeByte(IODIR, 0x00);
+      this.writeByte(this.IODIR, 0x00);
 
       // Write GPIO
-      this.writeByte(GPIO, state & 0xFF);
+      this.writeByte(this.GPIO, state & 0xFF);
 
       // Restore previous IODIR state
-      this.writeByte(IODIR, iodir);
+      this.writeByte(this.IODIR, iodir);
     }
   }
   bankRead() {
-    const { IODIR, GPIO } = this.reg;
-
     if (this.offset) {
       // Read IODIR state
-      let iodir = this.readWord(IODIR);
+      let iodir = this.readWord(this.IODIR);
 
       // Set IODIR state to INPUT
-      this.writeWord(IODIR, 0xFFFF);
+      this.writeWord(this.IODIR, 0xFFFF);
 
       // Read GPIO
-      let gpio = this.readWord(GPIO);
+      let gpio = this.readWord(this.GPIO);
 
       // Restore previous IODIR state
-      this.writeWord(IODIR, iodir);
+      this.writeWord(this.IODIR, iodir);
 
       return gpio;
     } else {
       // Read IODIR state
-      let iodir = this.readByte(IODIR);
+      let iodir = this.readByte(this.IODIR);
 
       // Set IODIR state to INPUT
-      this.writeByte(IODIR, 0xFF);
+      this.writeByte(this.IODIR, 0xFF);
 
       // Read GPIO
-      let gpio = this.readByte(GPIO);
+      let gpio = this.readByte(this.GPIO);
 
       // Restore previous IODIR state
-      this.writeByte(IODIR, iodir);
+      this.writeByte(this.IODIR, iodir);
```

Comment:
Updates per **Device specific constants** section of https://github.com/Moddable-OpenSource/moddable/pull/21#issuecomment-353265714